BaragonService/pom.xml

```diff
@@ -228,6 +228,10 @@
       <artifactId>jukito</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.hubspot</groupId>
+      <artifactId>BaragonStatePersister</artifactId>
+    </dependency>
```

Comment:
We might not need this to be a new module. We could stash the `BaragonStatePersister` interface in `BaragonCore`.

Reply:
Moved to BaragonCore - thanks!